File: C/Iniciante/1043triangulo.c
#include <stdio.h>
#include <math.h>
int main() {
    double a,b,c,calc,ab,ac,bc,aab,aac,abc;
    scanf("%lf%lf%lf",&a,&b,&c);
    ab=a+b;
    ac=a+c;
    bc=b+c;
    aab=fabs(a-b);
    aac=fabs(a-c);
    abc=fabs(b-c);
    if((abc<a && a<bc)&&(aac<b && b<ac)&&(aab<c && a<ab)){
        calc=a+b+c;
        printf("Perimetro = %.1lf\n",calc);
    }else{
        calc=((a+b)/2)*c;
        printf("Area = %.1lf\n",calc);
    }
    return 0;
}

File: Python/iniciante/1043triangulo.py
import math as m
linha = input().split(" ")
a, b, c = linha
ao = a = float(a)
bo = b = float(b)
co = c = float(c)
ab = a+b
ac = a+c
bc = b+c
aab = m.fabs(a-b)
aac = m.fabs(a-c)
abc = m.fabs(b-c)
if (abc < a < bc)and(aac < b < ac)and(aab < c and a < ab):
    calc = a+b+c
    print("Perimetro = %.1lf" % calc)
else:
    calc = ((a+b)/2)*c
    print("Area = %.1lf" % calc)
